### DISCUSSION TITLE
appimage does not start

### DISCUSSION THREAD

**Discussion by sudocode1** [ppy/osu#22610] [Q&A] [answered]
when trying to run the osu! appimage, i get the following error:
![image](https://user-images.githubusercontent.com/52894745/218268714-b862ae2c-7526-4462-a0e1-983c2bdc4d7a.png)

the window does not open - i just get the error shown above and nothing else.

im on arch (kernel 6.1.10-arch1-1), with gnome as de. if you need any more info i can provide.

here are my logs:
[database.log](https://github.com/ppy/osu/files/10714081/database.log)
[input.log](https://github.com/ppy/osu/files/10714082/input.log)
[network.log](https://github.com/ppy/osu/files/10714083/network.log)
[performance.log](https://github.com/ppy/osu/files/10714084/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10714085/runtime.log)

**Comment by bdach** [comment, marked as answer]
Please check https://github.com/ppy/osu/issues/10319 and https://github.com/ppy/osu/issues/12550
